opennlp-tools/src/main/java/opennlp/tools/ml/model/EvalParameters.java

```diff
@@ -87,4 +89,22 @@ public double getCorrectionParam() {
   public void setCorrectionParam(double correctionParam) {
     this.correctionParam = correctionParam;
   }
-}
\ No newline at end of file
+
+  @Override
+  public boolean equals(Object obj) {
+    if (obj == this) {
+      return true;
+    }
+    else if (obj instanceof EvalParameters) {
+      EvalParameters evalParameters = (EvalParameters) obj;
+      return Arrays.equals(params, evalParameters.params)
+              && numOutcomes == evalParameters.numOutcomes
+              && correctionConstant == evalParameters.correctionConstant
+              && constantInverse == evalParameters.constantInverse
+              && correctionParam == evalParameters.correctionParam;
+    }
+    else {
+      return false;
+    }
```

Comment:
Rewrite the above as  
`if (obj instanceOf EvalParmeters) { 
   ...
  return ...
}

return obj == this;

  
`